src/store/index.js

```diff
@@ -99,9 +93,16 @@ export default createStore({
 
 		selectedImages: (state) => state.selectedImages,
 
-		firstLargeImage: (state) => {
-			if (state.largeImageCache.length) {
-				return state.largeImageCache[0].url
+		// given an image basename, fetches the large res of that image
+		largeImage: (state) => (imageBasename) => {
```

Comment:
What happens when the Project View page first loads? I ask because I see that in `handleThumbnailClick` this getter is being used. But `handleThumbnailClick` is only invoked when clicking thumbnails

Reply:
at line 12 in project view when the component is created